libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -2355,6 +2358,8 @@ static CK_RV prvCreatePublicKey( CK_ATTRIBUTE * pxTemplate,
             /* coverity[misra_c_2012_rule_10_5_violation] */
             xResult = prvCreateECKey( pxTemplate, ulCount, pxObject, ( CK_BBOOL ) CK_FALSE );
         }
+    #else
```

Comment:
If you hit this preprocess else, the resulting code will have a statement between the If above and the else below which leads to a compiler error.